### PR TITLE
[IT-strings.xml] Minor fixes

### PR DESCRIPTION
**Pull request by hrumag**
Hi all!

I made some minor changes to strings.xml in order to fix some issues.
The most important, the erroneous '/n' displayed at fingerprint authentication.

Please review, @erciccione.

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -315,8 +315,8 @@
     <string name="tx_account">Account #</string>
 
     <string name="send_sweepall">Manda tutti i fondi confermati in questo account!</string>
-    <string name="tx_subaddress">Subaddress #%1$d</string>
-    <string name="generate_address_label_sub">Subaddress pubblico #%1$d</string>
+    <string name="tx_subaddress">Sottoindirizzo #%1$d</string>
```
Comment (erciccione) [inline app/src/main/res/values-it/strings.xml:318]: I would leave 'subadress' as it is for now. 'sottoindirizzo' could be confusing. I think this is a discussion for monero-ecosystem/monero-translations

**Review by erciccione — COMMENTED** [review]
Some arbitrary changes, but i'm ok with them. Only one comment about the translation of 'subadress'

File: app/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (hrumag) [reply app/src/main/res/values-it/strings.xml:318]: I reverted the latest commit in order to take the discussion to  monero-ecosystem/monero-translations.
The review can continue.

**Review by erciccione — APPROVED** [review]
Thanks @hrumag :)

**Comment by hrumag** [comment]
Considering, https://github.com/monero-ecosystem/monero-translations/issues/32, I've re-inserted "sottoindirizzo" translation and squashed all commits.

**Comment by erciccione** [comment]
Please rename the commit in something clearer

**Comment by hrumag** [comment]
@erciccione this PR does not seem to be approved yet. The merge is blocked right now.
Could you please handle it?

**Review by erciccione — APPROVED** [review]
@hrumag merge is blocked because required the approval of a maintainer, which is only @m2049r.
Can't do more than leave my approval :)

**Review by m2049r — APPROVED** [review]

**Pull request merged**